app/components/download-switch/download-switch.tsx

```diff
@@ -1,13 +1,10 @@
-import React, { useEffect, useState } from "react"
```

Comment:
I get an error on Android that `React` is missing because you removed it. Even though it's never used directly, on Android it needs to be imported in every component.

Reply:
Fixed in https://github.com/simonwacker/yoga-insights/pull/53/commits/ab61a0b91cd4297a2095a19e3fcaf8e8448f59e3